.github/workflows/trufflehog.yml

```
@@ -15,4 +15,7 @@ jobs:
         with:
           fetch-depth: 0
       - name: Secret Scanning
-        uses: trufflesecurity/trufflehog@main
\ No newline at end of file
+        uses: trufflesecurity/trufflehog@main
+        with:
+          # Skip scanning for JSONL files since they are datasets
+          extra_args: --debug --exclude-globs="**.jsonl"
```

This is a fix recommended by our infra team
```suggestion
          # exclude buggy postgres detector that is causing false positives and not relevant to our codebase
        extra_args: --results=verified,unknown --exclude-detectors=postgres
```